ts/src/PaymentAPI.ts

```diff
@@ -115,6 +119,18 @@ export class PaymentAPI {
         return this.makeRequest('POST', revertUri, request);
     }
 
+    /**
+     * Payment Highway Revert Pivo Transaction with amount
```

Comment:
Maybe it's not quite clear how to do full amount reversals vs partial reversals.